ResearchKit/ActiveTasks/ORKTrailmakingContentView.h

```diff
@@ -0,0 +1,53 @@
+/*
+ Copyright (c) 2016, Motus Design Group Inc. All rights reserved.
```

Comment:
Will Apple want these copyrights to say Apple, Inc. when this is submitted to their fork? (assuming it will be)